### ISSUE TITLE
About Curriculum Learning

### ISSUE THREAD

**Issue by Andy1621** [m-bain/frozen-in-time#42]
Thanks for your great job! Heer are some questions about Curriculum Learning.

When fine-tuning from 1 frame to 4 frames, 
- should we need to interpolate the temporal position embedding ([1, dim] => [4, dim])? In my opinion, the image is seen as 1-frame video, if the temporal position embedding is interpolated, how can we add it to the image?
- should we use the same hyperparameters (e.g., learning rate, epoch, warmup)?

**Comment by m-bain** [comment]
Hi,

Regarding the positinal embedding interpolation, we found that the method used doesn't make too much of a difference. See Table 7 in the supplementary https://openaccess.thecvf.com/content/ICCV2021/supplemental/Bain_Frozen_in_Time_ICCV_2021_supplemental.pdf

We suspect this is because the model doesn't really use the positional embeddings that much (they aren't super important for the video retrieval task).

When fine-tuning 1-frame to 4 frames yeah you can use the same learning rate and everything, 4-frame finetuning requires only ~10% of the number of epochs though.

**Comment by Andy1621** [comment]
Thanks for your reply! It really helps!

**Comment by Andy1621** [comment]
BTW, what does `Zero-pad` mean? 
When increasing the input number of frames from 4 to 8, how does `Zero-pad` work? 
The center 4 frames use the original temporal embedding, and the other frames use zero padding?
